lib/manageiq/loggers/json_logger.rb

```diff
@@ -0,0 +1,77 @@
+module ManageIQ
+  module Loggers
+    class JSONLogger < Base
```

Comment:
I named it this way on purpose to avoid it colliding with the toplevel JSON.  I originally had `ManageIQ::Loggers::JSON`, but was concerned that it would pollute the `ManageIQ::Loggers` namespace, and if a different logger needed JSON it would resolve to the wrong thing.